pom.xml

```diff
@@ -57,8 +57,8 @@
     <version.org.eclipse.jdt>3.19.0</version.org.eclipse.jdt>
     <version.com.github.javaparser>3.13.10</version.com.github.javaparser>
     <version.com.fasterxml.jackson>2.9.10</version.com.fasterxml.jackson>
-    <version.kubernetes-client>4.4.2</version.kubernetes-client>
-    <version.okhttp>3.12.0</version.okhttp>
+    <version.kubernetes-client>4.6.4</version.kubernetes-client>
```

Comment:
updating kubernetes-client to 4.6.4 requires an update of jackson
see https://github.com/kiegroup/kogito-bom/pull/76
We should put version 4.6.2

Reply:
@radtriste ack, thanks.